actions/exercise_actions.ts

```
@@ -6,326 +6,364 @@ import { desc, sql, and } from "drizzle-orm";
+        case "name":
+          return sql`${Exercise.exerciseName} ILIKE ${`%${value}%`}`;
+        case "motion":
+          return sql`${Exercise.motion} ILIKE ${`%${value}%`}`;
+        case "targetArea":
+          return sql`${Exercise.targetArea} ILIKE ${`%${value}%`}`;
+        case "status":
+          // For boolean filter, convert string to boolean
+          const boolValue =
+            value === "true" || value === "True" || value === "TRUE"
+              ? true
+              : false;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)